build/buildorder.props

```diff
@@ -29,6 +29,7 @@
     <RepositoryBuildOrder Include="MusicStore" Order="16" RootPath="$(RepositoryRoot)src\MusicStore\" />
     <RepositoryBuildOrder Include="SignalR" Order="16" />
     <RepositoryBuildOrder Include="AuthSamples" Order="16" RootPath="$(RepositoryRoot)src\AuthSamples\" />
-    <RepositoryBuildOrder Include="Templating" Order="17" RootPath="$(RepositoryRoot)src\Templating\" />
+    <RepositoryBuildOrder Include="Components" Order="17" RootPath="$(RepositoryRoot)src\Components\" />
```

Comment:
I assume this is here because Blazor relies on SignalR?

Reply:
Correct